### PR TITLE
Creating instructions to run travis build locally

### PR DESCRIPTION
**Pull request by ooq**

### CODE REVIEW

**Comment by ooq** [comment]
This documents the steps I use to fix #403.
@aysylu @nenaddedic

File: docs/debugging-travis.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+1. Find the latest `xenial` image tag at the travis dockerhub [repo](https://hub.docker.com/r/travisci/ci-sardonyx/tags). 
+As an example, the latest image tag as of writing is `packer-1564753982-0c06deb6`.
+
+```shell
+$ IMAGE="travisci/ci-sardonyx:packer-1564753982-0c06deb6"
+```
+2. Start a Docker container.
+
+```shell
+$ BUILDID="build-$RANDOM"
```
Comment (aysylu) [inline docs/debugging-travis.md]: How's the `RANDOM` picked here? just manually by the user?

Comment (ooq) [reply docs/debugging-travis.md]: This reuses the `RANDOM` system environment variable.

Comment (aysylu) [reply docs/debugging-travis.md]: Oh, gotcha!

**Comment by ooq** [comment]
@aysylu I don't have permission to merge this PR. Can you merge it? or let me know if there is any other pending actions.

**Comment by aysylu** [comment]
@ooq sure thing! all done, thanks!

**Pull request merged**